{"name": "Тест получения кода 404 при авторизации несуществующего курьера", "status": "passed", "description": "Проверка, что при запросе c несуществующей парой логин-пароль получим ошибку 404", "steps": [{"name": "POST запрос на ручку \"/api/v1/courier/login\" - авторизация курьера", "status": "passed", "parameters": [{"name": "data", "value": "{'login': 'sergeik14', 'password': '1234'}"}], "start": 1721555359784, "stop": 1721555359867}], "attachments": [{"name": "Response status", "source": "d270a265-b630-495d-99df-a19783bbfd77-attachment.attach"}], "start": 1721555359784, "stop": 1721555359869, "uuid": "b84223e4-2f8d-4cd6-9a9c-8e4e6079faef", "historyId": "b09ba69fbec36c7ea70053da51c3875a", "testCaseId": "b09ba69fbec36c7ea70053da51c3875a", "fullName": "tests.test_login_courier.TestLoginCourier#test_login_courier_not_existent_pair_login_pass_return_404", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "0000NBB0MJTQ6L4"}, {"name": "thread", "value": "89677-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}